csharp/serialization/Loan.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace serialization
+{
+    // <Snippet2>
+    [Serializable()]
+    // </Snippet2>
+    // <Snippet1>
+   public class Loan : INotifyPropertyChanged
+    {
+        public double LoanAmount { get; set; }
+        public double InterestRate { get; set; }
+
+        // <Snippet4>
+        [field:NonSerialized()]
+        public DateTime TimeLastLoaded { get; set; }
+        // </Snippet4>
+
+        public int Term { get; set; }
+
+        private string customer;
+        public string Customer
+        {
+            get { return customer; }
+            set
+            {
+                customer = value;
+                PropertyChanged(this,
```

Comment:
@BillWagner shouldn't this line be
```csharp
PropertyChanged?.Invoke(this,
```

Reply:
Oops, I've realized I can update it myself as it's already merged into the master